src/quo2/components/buttons/slide_button/style.cljs

```diff
@@ -0,0 +1,90 @@
+(ns quo2.components.buttons.slide-button.style
+  (:require
+   [quo2.components.buttons.slide-button.consts :as consts]
+   [quo2.components.buttons.slide-button.utils :as utils]
+   [react-native.reanimated :as reanimated]
+   [quo2.foundations.typography :as typography]))
+
+(def absolute-fill
+  {:position :absolute
+   :top      0
+   :bottom   0
+   :left     0
+   :right    0})
+
+(defn thumb-container
+  [interpolate-track thumb-size customization-color]
+  (reanimated/apply-animations-to-style
+   {:transform [{:translate-x (interpolate-track :track-clamp)}]}
+   {:background-color (utils/slider-color :main customization-color)
+    :border-radius 12
+    :height thumb-size
+    :width thumb-size
+    :align-items :center
+    :overflow :hidden
+    :justify-content :center}))
+
+(defn arrow-icon-container
+  [interpolate-track]
+  (reanimated/apply-animations-to-style
+   {:transform [{:translate-x (interpolate-track :arrow-icon-position)}]}
+   {:flex 1
+    :align-items :center
+    :justify-content :center}))
+
+(defn action-icon
+  [interpolate-track size]
+  (reanimated/apply-animations-to-style
+   {:transform [{:translate-x (interpolate-track :action-icon-position)}]}
+   {:height size
+    :width size
+    :position :absolute
+    :align-items :center
+    :left 0
+    :top 0
+    :z-index 2
+    :flex-direction :row
+    :justify-content :space-around}))
+
+(defn track-container
+  [height]
+  {:align-self       :stretch
+   :align-items      :center
+   :justify-content  :center
+   :height height})
+
+(defn track
+  [disabled? customization-color]
+  {:align-items      :flex-start
+   :justify-content  :center
+   :border-radius    14
+   :align-self       :stretch
+   :padding          consts/track-padding
+   :opacity          (if disabled? 0.3 1)
+   :background-color (utils/slider-color :track  customization-color)})
+
+(defn track-cover
+  [interpolate-track]
+  (reanimated/apply-animations-to-style
+   {:left (interpolate-track :track-cover)}
+   (merge {:overflow :hidden} absolute-fill)))
```

Comment:
Unless you think `absolute-fill` will override `:overflow :hidden` (if that's the case then the code is correct),
I'd swap the two and use `assoc`

`(assoc absolute-fill :overflow :hidden)`

but that won't allow `absolute-fill` to override `:overflow`, so depends on usage (either are completely fine)

Reply:
done